[2024-03-16T09:14:39.565+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: study_case_etl_cloud_storage.fetch_data_task manual__2024-03-16T09:14:37.574437+00:00 [queued]>
[2024-03-16T09:14:39.574+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: study_case_etl_cloud_storage.fetch_data_task manual__2024-03-16T09:14:37.574437+00:00 [queued]>
[2024-03-16T09:14:39.574+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-16T09:14:39.586+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): fetch_data_task> on 2024-03-16 09:14:37.574437+00:00
[2024-03-16T09:14:39.591+0000] {standard_task_runner.py:60} INFO - Started process 1697 to run task
[2024-03-16T09:14:39.594+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'study_case_etl_cloud_storage', 'fetch_data_task', 'manual__2024-03-16T09:14:37.574437+00:00', '--job-id', '702', '--raw', '--subdir', 'DAGS_FOLDER/etl_day_bigquery/study_case_etl_cloud_storage.py', '--cfg-path', '/tmp/tmpr0zc4r_6']
[2024-03-16T09:14:39.597+0000] {standard_task_runner.py:88} INFO - Job 702: Subtask fetch_data_task
[2024-03-16T09:14:39.633+0000] {task_command.py:423} INFO - Running <TaskInstance: study_case_etl_cloud_storage.fetch_data_task manual__2024-03-16T09:14:37.574437+00:00 [running]> on host 1d5b76b26ba5
[2024-03-16T09:14:39.702+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Agus mahari' AIRFLOW_CTX_DAG_ID='study_case_etl_cloud_storage' AIRFLOW_CTX_TASK_ID='fetch_data_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-16T09:14:37.574437+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-16T09:14:37.574437+00:00'
[2024-03-16T09:14:39.704+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:315: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-03-16T09:14:42.256+0000] {logging_mixin.py:188} INFO - 2024-03-16 09:14:42,256 INFO sqlalchemy.engine.Engine SELECT DATABASE()
[2024-03-16T09:14:42.256+0000] {base.py:1863} INFO - SELECT DATABASE()
[2024-03-16T09:14:42.257+0000] {logging_mixin.py:188} INFO - 2024-03-16 09:14:42,256 INFO sqlalchemy.engine.Engine [raw sql] {}
[2024-03-16T09:14:42.256+0000] {base.py:1868} INFO - [raw sql] {}
[2024-03-16T09:14:43.149+0000] {logging_mixin.py:188} INFO - 2024-03-16 09:14:43,149 INFO sqlalchemy.engine.Engine SELECT @@sql_mode
[2024-03-16T09:14:43.149+0000] {base.py:1863} INFO - SELECT @@sql_mode
[2024-03-16T09:14:43.150+0000] {logging_mixin.py:188} INFO - 2024-03-16 09:14:43,150 INFO sqlalchemy.engine.Engine [raw sql] {}
[2024-03-16T09:14:43.150+0000] {base.py:1868} INFO - [raw sql] {}
[2024-03-16T09:14:43.592+0000] {logging_mixin.py:188} INFO - 2024-03-16 09:14:43,591 INFO sqlalchemy.engine.Engine SELECT @@lower_case_table_names
[2024-03-16T09:14:43.591+0000] {base.py:1863} INFO - SELECT @@lower_case_table_names
[2024-03-16T09:14:43.592+0000] {logging_mixin.py:188} INFO - 2024-03-16 09:14:43,592 INFO sqlalchemy.engine.Engine [raw sql] {}
[2024-03-16T09:14:43.592+0000] {base.py:1868} INFO - [raw sql] {}
[2024-03-16T09:14:44.254+0000] {logging_mixin.py:188} INFO - 2024-03-16 09:14:44,254 INFO sqlalchemy.engine.Engine 


  SELECT 
    *
FROM databasename.mytable
[2024-03-16T09:14:44.254+0000] {base.py:1863} INFO - 


  SELECT 
    *
FROM databasename.mytable
[2024-03-16T09:14:44.255+0000] {logging_mixin.py:188} INFO - 2024-03-16 09:14:44,255 INFO sqlalchemy.engine.Engine [raw sql] {}
[2024-03-16T09:14:44.255+0000] {base.py:1868} INFO - [raw sql] {}
[2024-03-16T09:14:44.702+0000] {python.py:202} INFO - Done. Returned value was: /opt/***/data/aplikasi_table_2024-03-16-9-14-37.parquet
[2024-03-16T09:14:44.719+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=study_case_etl_cloud_storage, task_id=fetch_data_task, execution_date=20240316T091437, start_date=20240316T091439, end_date=20240316T091444
[2024-03-16T09:14:44.772+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-16T09:14:44.792+0000] {taskinstance.py:3309} INFO - 1 downstream tasks scheduled from follow-on schedule check
